examples/undocumented/python/kernel_combined.py

```diff
@@ -25,17 +24,17 @@ def kernel_combined (fm_train_real=traindat,fm_test_real=testdat,fm_train_dna=tr
 	feats_test.append_feature_obj(subkfeats_test)
 	kernel.add("kernel_array", subkernel)
 
-	subkfeats_train=StringCharFeatures(fm_train_dna, DNA)
-	subkfeats_test=StringCharFeatures(fm_test_dna, DNA)
+	subkfeats_train=sg.create_string_features(fm_train_dna, sg.DNA)
+	subkfeats_test=sg.create_string_features(fm_test_dna, sg.DNA)
 	degree=3
-	subkernel=FixedDegreeStringKernel(10, degree)
+	subkernel=sg.create_kernel("FixedDegreeStringKernel", degree=degree)
 	feats_train.append_feature_obj(subkfeats_train)
 	feats_test.append_feature_obj(subkfeats_test)
 	kernel.add("kernel_array", subkernel)
 
-	subkfeats_train=StringCharFeatures(fm_train_dna, DNA)
-	subkfeats_test=StringCharFeatures(fm_test_dna, DNA)
-	subkernel=LocalAlignmentStringKernel(10)
+	subkfeats_train=sg.create_string_features(fm_train_dna, sg.DNA)
+	subkfeats_test=sg.create_string_features(fm_test_dna, sg.DNA)
+	subkernel=sg.create_kernel("LocalAlignmentStringKernel")
 	feats_train.append_feature_obj(subkfeats_train)
```

Comment:
btw this should work via the new interface already via `add` ?

Reply:
True, another class that can be removed :)